core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -390,6 +390,45 @@ static HttpResponse of(Publisher<? extends HttpObject> publisher) {
         }
     }
 
+    /**
+     * Creates a new HTTP response of the redirect to specific location.
+     */
+    static HttpResponse ofRedirect(HttpStatus redirectStatus, String location) {
+        requireNonNull(redirectStatus, "redirectStatus");
+        requireNonNull(location, "location");
+        if (redirectStatus.compareTo(HttpStatus.MULTIPLE_CHOICES) < 0 ||
+            redirectStatus.compareTo(HttpStatus.TEMPORARY_REDIRECT) > 0) {
+            throw new IllegalArgumentException("redirectStatus: " + redirectStatus + " (expected: 300 .. 307)");
+        }
+
+        return of(ResponseHeaders.of(redirectStatus, HttpHeaderNames.LOCATION, location));
+    }
+
+    /**
+     * Creates a new HTTP response of the redirect to specific location using string format.
+     */
+    static HttpResponse ofRedirect(HttpStatus redirectStatus, String format, Object... args) {
+        requireNonNull(format, "format");
```

Comment:
Could also add `requireNonNull` for `args`

Reply:
Added `args` check and test cases more.